### PR TITLE
Fix naming of VTStor support Trim command task

### PR DESCRIPTION
**Pull request by minhvirtium**
- Edit naming convention
- Edit inclusion
- Edit reference of parameter

### CODE REVIEW

File: vtStor/DataCommandStructure.h [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-#define __DataCommandStructure_h__
-#pragma once
-
-#include <vector>
-
-#include "BasicTypes.h"
-
-namespace vtStor
-{
-    struct sLbaRangeEntry
```
Comment (minhvirtium) [inline vtStor/DataCommandStructure.h:29]: Moved Struct to Ata.h and deleted this file

File: vtStorAta/TrimBufferFormater.cpp [hunk 2]
```diff
@@ -45,9 +45,9 @@ namespace vtStor
             }
         }
 
-        void cTrimBufferFormater::FillWriteBufferWithLbaEntriesRange(std::shared_ptr<IBuffer>& DataBuffer, std::vector<sLbaRangeEntry> LbaRangeEntries)
+        void cTrimBufferFormater::FillWriteBufferWithLbaEntriesRange(std::shared_ptr<IBuffer>& DataBuffer, std::vector<StorageUtility::Ata::sLbaRangeEntry> LbaRangeEntries)
```
Comment (hai-tran) [inline vtStorAta/TrimBufferFormater.cpp]: TrimBufferFormatter

Comment (minhvirtium) [inline vtStorAta/TrimBufferFormater.cpp]: I will edit

File: vtStorAta/TrimBufferFormatter.cpp [hunk 3]
```diff
@@ -17,21 +17,21 @@ limitations under the License.
...
         {
         }
 
-        cTrimBufferFormater::~cTrimBufferFormater()
+        cTrimBufferFormatter::~cTrimBufferFormatter()
         {
         }
 
-        void cTrimBufferFormater::AddLbaRangeEntryToWriteBuffer(std::shared_ptr<IBuffer>& DataBuffer, U64 StartLba, U16 SectorCount, U32 LbaRangeEntryOffset)
+        void cTrimBufferFormatter::AddLbaRangeEntryToWriteBuffer(std::shared_ptr<IBuffer>& DataBuffer, U64 StartLba, U16 SectorCount, U32 LbaRangeEntryOffset)
```
Comment (hai-tran) [inline vtStorAta/TrimBufferFormatter.cpp]: AddLbaRangeEntryToBuffer()

File: vtStorAta/TrimBufferFormatter.cpp [hunk 4]
```diff
@@ -45,9 +45,9 @@ namespace vtStor
             }
         }
 
-        void cTrimBufferFormater::FillWriteBufferWithLbaEntriesRange(std::shared_ptr<IBuffer>& DataBuffer, std::vector<sLbaRangeEntry> LbaRangeEntries)
+        void cTrimBufferFormatter::FillWriteBufferWithLbaEntriesRange(std::shared_ptr<IBuffer>& DataBuffer,const std::vector<StorageUtility::Ata::sLbaRangeEntry>& LbaRangeEntries)
```
Comment (hai-tran) [inline vtStorAta/TrimBufferFormatter.cpp]: FillBufferWithLbaRangeEntries

Comment (minhvirtium) [inline vtStorAta/TrimBufferFormatter.cpp]: Done

File: vtStorAta/TrimBufferFormatter.cpp (hunk 3 above)
Comment (minhvirtium) [inline vtStorAta/TrimBufferFormatter.cpp]: Done

File: Common/DriveProperties.h [hunk 5]
```diff
@@ -15,22 +15,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 </License>
 */
-
-#ifndef __DataCommandStructure_h__
-#define __DataCommandStructure_h__
+#ifndef __DriveProperties_h__
```
Comment (tamvirtium) [inline Common/DriveProperties.h:18]: Why **DriveProperties_h** and thing related to its existed on your commit?

**Pull request merged**